soc/nordic/nrf54h/global_hsfll.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/clock_control/nrf_clock_control.h>
+
+static struct onoff_client cli;
+static const struct device *global_hsfll = DEVICE_DT_GET(DT_NODELABEL(hsfll120));
+static const struct nrf_clock_spec spec = {
+	.frequency = CONFIG_SOC_NRF54H20_GDFS_MIN_FREQ,
```

Comment:
nit: This file is created in the wrong commit, where it's in an unfinished state.